app/src/main/java/com/yujeong/aostudy/broccoli/ui/home/HomeFragment.kt

```diff
@@ -0,0 +1,56 @@
+package com.yujeong.aostudy.broccoli.ui.home
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.google.android.material.badge.BadgeDrawable
+import com.google.android.material.tabs.TabLayoutMediator
+import com.yujeong.aostudy.broccoli.R
+import com.yujeong.aostudy.broccoli.databinding.FragmentHomeBinding
+import com.yujeong.aostudy.broccoli.ui.base.BaseFragment
+import com.yujeong.aostudy.broccoli.ui.home.*
+import com.yujeong.aostudy.broccoli.util.PagerFragmentStateAdapter
+
+class HomeFragment : BaseFragment<FragmentHomeBinding>(R.layout.fragment_home){
+
+    private lateinit var mContext: Context
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        super.onCreateView(inflater, container, savedInstanceState)
+        mContext = requireContext()
+        
+        initPager()
+
+        return binding.root
+    }
+
+    private fun initPager(){
+
+        val pagerAdapter = PagerFragmentStateAdapter(requireActivity())
+        pagerAdapter.addFragment(RecommendHomeFragment())
+        pagerAdapter.addFragment(DomesticHomeFragment())
+        pagerAdapter.addFragment(EnjoyHomeFragment())
+        pagerAdapter.addFragment(TrafficHomeFragment())
+        pagerAdapter.addFragment(ForeignHomeFragment())
```

Comment:
```suggestion
        pagerAdapter.addFragment(EmptyFragment())
        pagerAdapter.addFragment(EmptyFragment())
        pagerAdapter.addFragment(EmptyFragment())
        pagerAdapter.addFragment(EmptyFragment())
        pagerAdapter.addFragment(EmptyFragment())
```
만약에 더 구현하지 않을 거면 이렇게 해줘도 됩니당 :)